rel/commands/sync.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+release="_build/dev/rel/mana/bin/mana"
+$release eval --mfa "Mix.Tasks.Sync.run/1" --argv -- "$@"
```

Comment:
Not super familiar with `distillery`, but I was perusing the docs. Could we use `release_ctl` here instead of the combination `release="_build/dev/rel/mana/bin/mana"` and `$release`? 

See [shell scripts](https://hexdocs.pm/distillery/extensibility/shell_scripts.html) and an example of a [custom command](https://hexdocs.pm/distillery/guides/running_migrations.html#custom-command)